lib/models/session.rb

```diff
@@ -0,0 +1,14 @@
+# typed: strict
+# frozen_string_literal: true
+
+module AutoHCK
+  module Models
+    class Session < T::Struct
+      extend T::Sig
+      extend JsonHelper
+
+      const :test, AutoHCK::CLI::TestOptions
+      const :common, AutoHCK::CLI::CommonOptions
+    end
+  end
+end
```

Comment:
What about letting `AutoHCK::CLI` inherit `T::Struct` so that we can remove this class?